aseba/clients/CMakeLists.txt

```diff
@@ -28,17 +23,10 @@ if(NOT ANDROID)
 
 
     endif()
-
-    find_package(SDL2)
-    if (SDL2_FOUND)
-    add_subdirectory(joy)
-    endif()
-
-
```

Comment:
joy I am not sure, could be interesting to keep

Reply:
We can we can rewrite it with the new api ( once it supports events ) - maybe even in js, it would make a nice demo !